spec/promenade/client/rack/collector_spec.rb

```diff
@@ -1,164 +0,0 @@
-require "spec_helper"
-require "promenade/client/rack/collector"
-require "rack/mock"
-require "support/test_rack_app"
-
-RSpec.describe Promenade::Client::Rack::Collector, reset_prometheus_client: true do
-  describe "#call" do
-    it "preserves the status code" do
-      env = Rack::MockRequest.env_for
-      app = TestRackApp.new(status: 418)
-      middleware = Promenade::Client::Rack::Collector.new(app)
-      status, = middleware.call(env)
-
-      expect(status).to eql(418)
-    end
```

Comment:
I think these tests are still valid to have end to end testing

Reply:
Yep, I accidentally removed the file. Have re-added it this afternoon, but that was when I realised the implementation caused a lot of tests to break.